airflow/bin/cli.py

```diff
@@ -20,1703 +20,38 @@
 """Command-line interface"""
 
 import argparse
-import errno
-import functools
-import getpass
-import importlib
-import json
-import logging
 import os
-import random
-import re
-import reprlib
-import signal
-import string
-import subprocess
-import sys
 import textwrap
-import threading
-import time
-import traceback
 from argparse import RawTextHelpFormatter
-from importlib import import_module
-from typing import Any
-from urllib.parse import urlunparse
+from typing import Callable
 
-import daemon
-import psutil
-from daemon.pidfile import TimeoutPIDLockFile
-from sqlalchemy.orm import exc
-from tabulate import tabulate, tabulate_formats
+from tabulate import tabulate_formats
```

Comment:
This import seems unused now?

Reply:
https://github.com/apache/airflow/blob/911910e6feb2c04d7a0a69692832e0cc7e184c3e/airflow/bin/cli.py#L121-L128
It is the ways to display the table